ruby/ql/lib/change-notes/2022-09-23-yaml-load-file.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added `YAML.load_file` as a potential sink for unsafe deserialization.
```

Comment:
This needs to be changed now.

Reply:
Thanks, this slipped my mind. I've dropped the change note entirely as I don't think this change warrants one.